<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>948</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="myButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="myCheckBox">
     <property name="text">
      <string>CheckBox</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="myComboBox">
     <item>
      <property name="text">
       <string>kani</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tako</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ika</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="myRadioButton1">
        <property name="text">
         <string>rb1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="myRadioButton2">
        <property name="text">
         <string>rb2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="myLineEdit"/>
   </item>
   <item>
    <widget class="QTextEdit" name="myTextEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="myLabel3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="mySpinBox"/>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="myDoubleSpinBox"/>
   </item>
   <item>
    <widget class="QTreeWidget" name="myTreeWidget">
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>ika</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tako</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kani</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ika</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ebi</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="myTableWidget">
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>新しい行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>uni</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新しい行</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>ebi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新しい列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新しい列</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>ika</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>uniebi</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>fasdasdf</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>tako</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>dd</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>ika</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>kani</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="myVerticalSlider">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="myHorizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
